src/renderer/components/general-settings/general-settings.vue

```diff
@@ -9,17 +9,18 @@
     <ft-flex-box class="generalSettingsFlexBox">
       <ft-toggle-switch
         label="Fallback to Non-Preferred Backend on Failure"
+        :compact=false
```

Comment:
If false is the default for `compact` then this line can be deleted.  The component will know to set it to false.